src/azure/storage/loader.rs

```diff
@@ -77,6 +59,17 @@ impl Loader {
             return Ok(Some(cred));
         }
 
-        Ok(None)
+        // try to load credential using AAD(Azure Active Directory) authenticate on Azure VM
+        // we may get an error if not running on Azure VM
+        // see https://learn.microsoft.com/en-us/azure/app-service/overview-managed-identity?tabs=portal,http#using-the-rest-protocol
+        self.load_via_imds().await
```

Comment:
I'm guessing this call should in `load_inner`?

Reply:
Thank you for the quick reply, @Xuanwo. I have updated it.